libs/adsp-feedback/src/main.ts

```diff
@@ -463,6 +463,9 @@ class AdspFeedback implements AdspFeedbackApi {
     if (this.radio2Ref.value && this.radio2Ref.value.checked === true) {
       this.sendButtonRef.value?.removeAttribute('disabled');
     }
+    if (this.radio2Ref.value?.checked === false && this.radio1Ref.value?.checked === true) {
```

Comment:
This would be much easier to understand if the variable names reflected what actually was being checked instead of radio1 and radio2.  e.g. name them something like "ratingSelector" and "commentSelector". (I know that wasn't your choice, but "always leave the code better than you found it" is a good maxim to code by).

Reply:
Changed as "ratingSelector" and "commentSelector" instead of radio1 and radio2, for better understanding of variable names